R/count-department-diabetes-diagnoses.R

```diff
@@ -0,0 +1,27 @@
+#' Count diabetes-related primary diagnoses by department
+#'
+#' @param df A data.frame or tibble with the columns: is_t1d, is_t2d, is_primary, is_endo_department
+#' @return The same tibble grouped by pnr with added counts for T1D and T2D in different departments
+count_primary_diagnoses_by_department <- function(df) {
+  df |>
```

Comment:
```suggestion
count_primary_diagnoses_by_department <- function(data) {
  data |>
```

To be consistent with all our other functions.